client/src/pages/home/Home.jsx

```diff
@@ -47,10 +47,10 @@ function Home() {
   // States for Search.jsx
   const [SortOption, SwitchSortOption] = useState(false);
   const [FilterOption, SwitchFilterOption] = useState(false);
-  const [SortByVal, ChangeSortBy] = useState("");
+  const [setSortBy, ChangeSortBy] = useState("");
 
   //States for FilterSearch.jsx
-  const [locationVal, locationChange] = useState("");
+  const [setLocation, locationChange] = useState("");
   const [feeValue, changeFee] = useState([0, 500]);
```

Comment:
When using state the first return should be a variable name like `location` and `sortBy`, don't put set in it. Make the second return name consistent like `changeSoryBy` or `sortBychange`. I will recommend sticking to `setSortby` and `setLocation`

Reply:
Yes, this code was written before your last pull request comments which is why the naming is everywhere. I am using that convention everywhere now.